docs/configuration.asciidoc

```diff
@@ -46,44 +48,42 @@ public class Startup
 }
 ----
 
-NOTE: As explained in <<setup-asp-net-core>>, the `UseElasticApm` method only turns on ASP.NET Core monitoring. If you want to turn on everything (including HTTP and Database monitoring) you can use the `UseAllElasticApm` method from the `Elastic.Apm.NetCoreAll` package, which has the same overloads and behavior, except it turns on tracing for everything that is supported by the agent on .NET Core including database and HTTP tracing.
-
 With this you can use any configuration source that you configured on the `IConfiguration` instance that you passed to the APM Agent.
```

Comment:
This sentence doesn't make sense to me. What does "configuration source" reference?

Reply:
> This sentence doesn't make sense to me. What does "configuration source" reference?

[This doc from Microsoft](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-3.1) explains it - the term "configuration source" also comes from that doc. The config source can be a setting file, but also environment variables, or also some custom thing, or some cloud specific config thing, etc.

In practice this means agent config values can come from any of those sources. The idea here is that the agent is configured the same way as any other library, so the intention for doing this is better integration.

Can we maybe just link to that Microsoft doc? Or do we try to avoid doing so, because that doc is not under our control and could be moved?

Reply:
Hah, that's the same doc I was reading earlier, but I must've missed that section. I think we should definitely link to that file.

Reply:
Added in a link to the documentation!